integration_tests/run_pyspark_from_build.sh

```diff
@@ -424,6 +425,20 @@ else
             "${SPARK_HOME}"/bin/spark-shell "${SPARK_SHELL_ARGS_ARR[@]}" 2>/dev/null \
             | grep -F 'res0: Array[org.apache.spark.sql.Row] = Array([4950])'
         echo "SUCCESS spark-shell smoke test"
+    elif [[ "${EXPLAIN_ONLY_CPU_SMOKE_TEST}" != "0" ]]; then
+        echo "Running explainOnly mode on CPU smoke test..."
+        SPARK_SHELL_ARGS_ARR=(
+            --master local-cluster[1,2,1024]
```

Comment:
The error will be more trivially visible in the local mode 
```suggestion
            --master local[2]
```